src/cmd-prune

```diff
@@ -11,21 +11,17 @@ dn=$(dirname "$0")
 print_help() {
     cat 1>&2 <<'EOF'
 Usage: coreos-assembler prune --help
-       coreos-assembler prune [--keep=N] [--timestamp-only]
+       coreos-assembler prune [--keep=N]
```

Comment:
Good bye `--timestamp-only`!  We hardly knew ye!

Note, when this lands we'll need a corresponding change in the pipeline to stop using this command + flag.  (RHT-only link:  https://url.corp.redhat.com/406260f)

Reply:
Yup indeed! (Though only when it makes it to the `rhcos` branch).